data-model/digitalobjects/0.2.0/digital-specimens/schema/occurrences.json

```diff
@@ -0,0 +1,251 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/digitalobjects/0.2.0/digital-specimens/occurrences.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment":"DigitalObject Version 0.2.0",
+  "type": "object",
+  "properties": {
+    "dwc:organismQuantity": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/organismQuantity",
+      "examples": [
+        "27"
+      ]
+    },
+    "dwc:organismQuantityType": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/organismQuantityType",
+      "examples": [
+        "individuals"
+      ],
+      "$comment": "Could this become a enum?"
```

Comment:
we need to bring this up to dwc working group. I also see this in the desc "A dwc:organismQuantityType must have a corresponding dwc:organismQuantity. This term has an equivalent in the dwciri: namespace that allows only an IRI as a value, whereas this term allows for any string literal value."